polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs

```diff
@@ -182,6 +182,17 @@ impl BenchCli {
 	}
 }
 
+#[cfg(feature = "memprofile")]
+#[global_allocator]
+static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
+
+#[cfg(feature = "memprofile")]
+#[allow(non_upper_case_globals)]
+#[export_name = "malloc_conf"]
+// See https://jemalloc.net/jemalloc.3.html for more information on the configuration options.
+pub static malloc_conf: &[u8] =
+	b"prof:true,prof_active:true,lg_prof_interval:30,lg_prof_sample:21,prof_prefix:/tmp/subsystem-bench\0";
```

Comment:
I've use some of these options some time ago: https://gist.github.com/ordian/928dc2bd45022cddd547528f64db9174#gistcomment-4807555
but configured them via an env variable. This looks nice :)